### PR TITLE
Add simple example that falls in line with README

### PR DESCRIPTION
**Pull request by cgyurgyik**
This adds an example to the main FuTIL repository, a part of #215 .
This code is a quick snapshot of FuTIL components, and is used mostly so that the code in the README at https://github.com/cucapra/futil#compiler-development works.

### CODE REVIEW

**Comment by rachitnigam** [comment]
Perhaps we can link it from the README or something? Writing a short blurb under "Using the Compiler" would be good.

**Review by rachitnigam — APPROVED** [review]

**Comment by cgyurgyik** [comment]
> Perhaps we can link it from the README or something? Writing a short blurb under "Using the Compiler" would be good.

Done, PTAL. I'll add a bit about ```fud``` once its been merged.

**Comment by rachitnigam** [comment]
LGTM! Hit the big green button when tests pass.

**Pull request merged**
